include/sys/dsl_scan.h

```diff
@@ -107,7 +110,9 @@ typedef enum dsl_scan_flags {
 typedef struct dsl_scan {
 	struct dsl_pool *scn_dp;
 
-	boolean_t scn_pausing;
+	uint8_t scn_scrub_state;
+	uint64_t scn_scrub_pause_time;
+	boolean_t scn_suspending;
```

Comment:
It would be better to move `boolean_t scn_suspending` before `uint64_t scn_scrub_pause_time` to make sure this structure is nicely packed.